modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,269 @@
+// CNF-950 4.7 Installing the operators
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-installing-the-operators_{context}"]
```

Comment:
Can we just mention what operators we need and reference their installing instructions? Here we are duplicating content

Reply:
In the review, it was agreed not to use links because currently the Pantheon build machine won't allow cross-links to internal docs. Also, the modules in other docs are not granular enough to allow this. Let me know if you're ok with this.